test/parser_test.py

```diff
@@ -30,8 +30,7 @@ def test_real_save():
     from stellarisdashboard import cli, config
     from pathlib import Path
 
-    output_db = Path(
-        f"{config.CONFIG.base_output_path}/db/nexitronawareness_1329922464.db"
-    )
     cli.f_parse_saves(save_path="test/saves")
-    output_db.unlink(missing_ok=True)
+    output_db_dir = Path(f"{config.CONFIG.base_output_path}/db")
+    for db in list(output_db_dir.glob('*.db')):
+      db.unlink(missing_ok=True)
```

Comment:
I think this risks deleting db files from real games when running tests which I think we should avoid. 

Instead, we can use the tmp_path fixture [provided by pytest](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html) and let the OS handle the cleanup. Then, I would also parametrize the test to a test case per game ID so we can easier figure out what broke

It should work like this:

```
@pytest.mark.parametrize(
    "save_dir",
    [path for path in pathlib.Path(__file__).parent.glob(r"saves/*")],
    ids=lambda path: path.stem,
)
def test_real_save(save_dir, tmp_path):
    from stellarisdashboard import cli, config

    debug = config.CONFIG.debug_mode
    base_path = config.CONFIG.base_output_path

    config.CONFIG.debug_mode = True
    config.CONFIG.base_output_path = tmp_path
    try:
        cli.f_parse_saves(save_path=save_dir)
    finally:
        config.CONFIG.debug_mode = debug
        config.CONFIG.base_output_path = base_path
```

Looks like this (before merging https://github.com/eliasdoehne/stellaris-dashboard/pull/128):
![image](https://github.com/eliasdoehne/stellaris-dashboard/assets/3602589/353005ea-b580-4023-9642-c3a7ae58fdfd)

Reply:
(maybe it also makes sense to organize the save files by game version, e.g. `test/saves/3.10.4/kaanvisamfriendship3_-449354374/2508.07.01.sav`)

Reply:
Good point, thank you. Updated